src/ops/cli/terraform.py

```diff
@@ -259,7 +259,8 @@ def run_v2_composition(self, args, terraform_path, composition):
         checkout_repo(
```

Comment:
add a log message before saying what it's happening below.

Reply:
There is already logging inside the function.